flexmeasures/ui/templates/crud/asset.html

```diff
@@ -236,6 +236,36 @@ <h3>All child assets for {{ asset.name }}</h3>
                 </table>
             </div>
         </div>
+        <div class="row">
+            <div class="copy-url" title="Click to copy URL to clipboard">
+                <script>
+                    function copyUrl() {
+                        if (!window.getSelection) {
+                            alert('Please copy the URL from the location bar.');
+                            return;
+                        }
+                        const dummy = document.createElement('p');
+                        dummy.textContent = window.location.href;
```

Comment:
The idea is to copy a URL representing what is shown in the plots, not the URL.
If I change the time range with the date selector, that should be the URL I'm copying. So I'm afraid we have to construct it here, not just copy the URL bar content.